components/net/fetch/methods.rs

```diff
@@ -673,8 +673,6 @@ fn http_redirect_fetch(request: Rc<Request>,
     request.redirect_count.set(request.redirect_count.get() + 1);
 
     // Step 9
```

Comment:
Step annotations for this fetch function needs to be updated to match the spec, I'm afraid.